inception-project/inception
# #1067 - Creating relations between named entities crashes

**What's in the PR**
- Improve parsing of relation endpoints
- Improve checking for synthetic annotations and rejecting the creation of relations on them

**How to test manually**
* See issue description
* Note this requires https://github.com/webanno/webanno/pull/1273 to be merged first

**Automatic testing**

**Documentation**
